CONTRIBUTING.md

```diff
@@ -1,7 +1,57 @@
-# How to Contribute
+# Contributing
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+We'd love to accept your patches and contributions to this project.
+
+## Development Environment
+
+If you are contributing documentation (rather than samples) and want to be able
+to view it in your browser, the easiest way to do so is to run the provided
+development server.
+
+We use [GitHub Pages][1] to make this documentation available, which uses
+[Jekyll][2] under the hood.
+
+If you have [Docker][3] installed, simply clone this repository and run the
```

Comment:
Minor: I'd like to remove the word "simply" here. [This](https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher) more-or-less explains why.

Reply:
Done.